packages/block-editor/src/components/inspector-controls-tabs/use-inspector-controls-tabs.js

```diff
@@ -42,15 +42,13 @@ export default function useInspectorControlsTabs( blockName ) {
 		color: colorGroup,
 		default: defaultGroup,
 		dimensions: dimensionsGroup,
-		list: listGroup,
 		typography: typographyGroup,
 	} = InspectorControlsGroups;
 
 	// List View Tab: If there are any fills for the list group add that tab.
 	const listViewDisabled = useIsListViewTabDisabled( blockName );
-	const listFills = useSlotFills( listGroup.Slot.__unstableName );
 
-	if ( ! listViewDisabled && !! listFills && listFills.length ) {
```

Comment:
I don't follow these changes. Why did they need to happen?

Reply:
The check ` && !! listFills && listFills.length` made the list view tab unavailable untill the fills loaded. I have no idea what this achieved, but removing this check allows the state to be 'list` and to set the default.

With the check in place the tabs reverted to 'styles' despite the state for initial tab was 'list' and that happened because of waiting for fills.

The other line is just removed because only the check used that const.

Reply:
I know @aaronrobertshaw worked on some of this so tagging him for additional context.

Reply:
From memory, the check was in place due to an earlier requirement that tabs were only to be rendered if they had fills. Given the list view tab is a special case using an allow list, it looks superfluous to me as well.